earthly/python/Earthfile

```diff
@@ -23,22 +27,30 @@ python-base:
     # Stop python code running in the container saving pointless .pyc files.
     ENV PYTHONDONTWRITEBYTECODE=1
 
-    # Install Poetry to manage python.
+    # Install Poetry using the recommended installer.
     RUN curl -sSL https://install.python-poetry.org | python3 -
-    # Prevent warning when running on machines with more than 6 cores.
-    #   "Connection pool is full, discarding connection: pypi.org. Connection pool size: 10"
+
+    # Adjust Poetry's configuration to prevent connection pool warnings.
     RUN poetry config installer.max-workers 10
 
-# Common python builder setup.
-# Parameters:
-#   opts      : Options passed to the `poetry install` command.
+    # Install ruff for linting instead of flake8.
+    RUN pip3 install ruff
+
+# Utilize the POETRY_SETUP UDC for setting up Poetry within our environment.
+poetry:
```

Comment:
Why do we need this?  It doesn't add anything over `python-base` itself?

Reply:
Deleted commit 3aeedb2559301de40cfccd457d699301e11235dc